homework2/task4/tests/tests_cache_function.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from homework2.task4.cache_function.cache_function import cache
+
+
+@pytest.fixture
+def test_add_finction():
+    def f_add(a, b):
+        return a + b
+
+    return f_add
+
+
+def test_cache_true(test_add_finction):
+    cache_true = cache(test_add_finction)
+    first_start = cache_true(10, 10)
+    second_start = cache_true(10, 10)
+    assert first_start == second_start
```

Comment:
this doesn't prove that result is cashed, only that result is equal

 
![image](https://user-images.githubusercontent.com/58084733/124377363-3e288900-dcb4-11eb-9aef-b0b3d5ca568f.png)